### DISCUSSION TITLE
A dam map

### DISCUSSION THREAD

**Discussion by lambda-11235** [redeclipse/base#1156] [Maps and Content Showcase]
It's a map with a dam and bridge in it.
![20181228184751](https://user-images.githubusercontent.com/16354815/50532671-7c380700-0ad1-11e9-8a81-ed5ee548a4fd.png)
![20181228184803](https://user-images.githubusercontent.com/16354815/50532672-7fcb8e00-0ad1-11e9-9da4-4f74fc565f71.png)
![20181228184813](https://user-images.githubusercontent.com/16354815/50532674-82c67e80-0ad1-11e9-8bfc-14ff42a9452b.png)

Data files: Note that these are also available on my [community](https://github.com/red-eclipse/community/tree/master/lambda_11235) subdirectory.

v0.1:
[dam.zip](https://github.com/red-eclipse/world/files/2682968/dam.zip)
[textures.zip](https://github.com/red-eclipse/world/files/2682970/textures.zip)

v0.2: Smooth dam walls.
[dam.zip](https://github.com/red-eclipse/world/files/2683157/dam.zip)

v0.3: Improve texturing, rework side route for better flow, add arches to bridge.
[dam.zip](https://github.com/red-eclipse/world/files/2683841/dam.zip)

v0.4: Smooth natural arch and add some pipes to dam.
[dam.zip](https://github.com/red-eclipse/world/files/2685066/dam.zip)

v0.5: Add secret passage and wrecked ship.
[dam.zip](https://github.com/red-eclipse/world/files/2692913/dam.zip)

v0.6: Use jump pad models instead of flat textures.
[dam.zip](https://github.com/red-eclipse/world/files/2696367/dam.zip)

v0.7: Change canyon texture and add more models.
[dam.zip](https://github.com/red-eclipse/world/files/2716012/dam.zip)

v0.7.1: Fix minor octree and texturing issues.
[dam.zip](https://github.com/red-eclipse/world/files/2719940/dam.zip)

The map uses one custom texture, included in textures.zip. I decided to use a custom texture because I thought the outdoor textures currently included didn't look good. If anyone has an idea for a better texture let me know. EDIT: The texture has been replaced. Now (0.7+) the map only depends on textures from base.

Also, any ideas on how to improve the map's visual appeal or balance the gameplay are welcome.

Edit: Everything is CC BY-SA by the way.

**Comment by no-lex** [comment]
https://www.deviantart.com/yughues/gallery/ is nobiax's gallery, with a lot of really good freely licensed high-def organic textures if that's what you're looking for.

I'll see about reviewing the map in a little bit.

**Comment by no-lex** [comment]
Some observations on this map, in no particular order:

- The map did not come remipped; make sure to calclight before a final release.
- Make sure to texturecull before making a final release; this chops about 200kb off of the final map size.
![screenshot from 2018-12-15 17-43-10](https://user-images.githubusercontent.com/23445929/50049107-581c0500-0091-11e9-9076-eca3a2e8cb5f.png)
- This jump is just a little too high to double-jump even while unencumbered, and breaks up the flow of the map for no reason.
![screenshot from 2018-12-15 17-43-35](https://user-images.githubusercontent.com/23445929/50049109-581c0500-0091-11e9-9a38-cf546799aaba.png)
- This is really gray, and gray is really boring. Using textures to break up the concrete greyness would make this look more immersing.
![screenshot from 2018-12-15 17-44-38](https://user-images.githubusercontent.com/23445929/50049110-581c0500-0091-11e9-80c5-e18f769fe2e5.png)
- Looks like a simple texturing error here. Maybe this can be changed into an arch to break up the cube-ness of the map, but either way is fine.
![screenshot from 2018-12-15 17-45-30](https://user-images.githubusercontent.com/23445929/50049111-581c0500-0091-11e9-85ea-6f0c4e47cf6d.png)
- Signs of clear computer-aided deoptimization here, probably by texturing with too small of a gridscale. Easily fixable, but something to watch out for at all times.

That being said, this is a better map than my first map (1 texture(!), trak/base(!), in three different tones ) that I submitted, and better than most first map attempts that I have seen. Mapping is hard, and especially given the constraints of the Cube geometry system, takes a lot of time to master, so keep at it.

**Comment by lambda-11235** [comment]
I added a lot of models in 0.7. If anyone could tell me if it negatively affects performance on their system that would be great.
